packages/ag-charts-website/src/components/framework-selector-inside-doc/FrameworkSelectorInsideDocs.tsx

```diff
@@ -0,0 +1,43 @@
+import { FRAMEWORKS } from '@constants';
+import fwLogos from '@images/fw-logos';
+import classnames from 'classnames';
+
+import { getNewFrameworkPath } from '../../utils/framework';
+import styles from './FrameworkSelectorInsideDocs.module.scss';
+
+export const FrameworkSelectorInsideDocs = ({ path, currentFramework }) => {
+    const handleFrameworkChange = (selectedFramework) => {
+        const newUrl = getNewFrameworkPath({
+            path,
+            currentFramework,
+            newFramework: selectedFramework,
+        });
+
+        window.location.href = newUrl;
+    };
+
+    const currentFrameworkLogo = currentFramework ? fwLogos[currentFramework] : null;
+
+    return (
+        <div className={classnames(styles.frameworkSelector)}>
+            <div className={styles.selectFrameworkContainer}>
+                {currentFrameworkLogo && (
+                    <img src={currentFrameworkLogo} alt={`${currentFramework} logo`} className={styles.frameworkLogo} />
+                )}
+                <span classnames={styles.divider}></span>
+                <select
+                    value={currentFramework}
+                    onChange={(event) => handleFrameworkChange(event.target.value)}
+                    onClick={(event) => event.stopPropagation()} // Prevent event propagation
+                    className={styles.select}
+                >
+                    {FRAMEWORKS.map((framework) => (
+                        <option value={framework} key={framework}>
+                            {framework.charAt(0).toUpperCase() + framework.slice(1)}
```

Comment:
You can use `getFrameworkDisplayText` from `@utils/framework` here, which does a similar thing, but we can change it all in one place